streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -201,6 +229,25 @@ public void shouldFetchAndIterateOverExactKeys() throws Exception {
 
         final List<KeyValue<Long, String>> expected = Utils.mkList(KeyValue.pair(0L, "0001"), KeyValue.pair(1L, "0003"), KeyValue.pair(60000L, "0005"));
         assertThat(toList(cachingStore.fetch("a", 0, Long.MAX_VALUE)), equalTo(expected));
+
+        assertThat(
```

Comment:
Same here. Would prefer to see at least 2 tests, one for each of the `fetch` methods